# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1483816509 18000
#      Sat Jan 07 14:15:09 2017 -0500
# Node ID 0f22914be30ed8eb735477534c9539f61a05eee4
# Parent  a2fc803e9556113f03f1c25b88f1493e557283a5
* acinclude.m4 (OCTAVE_CHECK_LIB): Downcase --with-XYZ options.

diff --git a/m4/acinclude.m4 b/m4/acinclude.m4
--- a/m4/acinclude.m4
+++ b/m4/acinclude.m4
@@ -645,46 +645,46 @@ AC_DEFUN([OCTAVE_CHECK_HDF5_HAS_VER_16_A
 ])
 dnl
 dnl Usage:
 dnl OCTAVE_CHECK_LIB(LIBRARY, DOC-NAME, WARN-MSG, HEADER, FUNC,
 dnl                  LANG, DOC-STRING, EXTRA-CHECK, PKG-CONFIG-NAME,
 dnl                  REQUIRED)
 dnl
 AC_DEFUN([OCTAVE_CHECK_LIB], [
-  AC_ARG_WITH([$1-includedir],
-    [AS_HELP_STRING([--with-$1-includedir=DIR],
+  AC_ARG_WITH([m4_tolower($1)-includedir],
+    [AS_HELP_STRING([--with-m4_tolower($1)-includedir=DIR],
       [look for $2 include files in DIR])],
     [m4_toupper([$1])_CPPFLAGS="-I$withval"])
   AC_SUBST(m4_toupper([$1])_CPPFLAGS)
 
-  AC_ARG_WITH([$1-libdir],
-    [AS_HELP_STRING([--with-$1-libdir=DIR],
+  AC_ARG_WITH([m4_tolower($1)-libdir],
+    [AS_HELP_STRING([--with-m4_tolower($1)-libdir=DIR],
       [look for $2 libraries in DIR])],
     [m4_toupper([$1])_LDFLAGS="-L$withval"])
   AC_SUBST(m4_toupper([$1])_LDFLAGS)
 
-  AC_ARG_WITH([$1],
+  AC_ARG_WITH([m4_tolower($1)],
     [ifelse([$#], 10,
        [m4_ifblank([$7],
-         [AS_HELP_STRING([--with-$1=<lib>], [use $2 library <lib>])],
-         [AS_HELP_STRING([--with-$1], [$7])])],
+         [AS_HELP_STRING([--with-m4_tolower($1)=<lib>], [use $2 library <lib>])],
+         [AS_HELP_STRING([--with-m4_tolower($1)], [$7])])],
        [m4_ifblank([$7],
-         [AS_HELP_STRING([--without-$1], [don't use $2 library])],
-         [AS_HELP_STRING([--without-$1], [$7])])])],
+         [AS_HELP_STRING([--without-m4_tolower($1)], [don't use $2 library])],
+         [AS_HELP_STRING([--without-m4_tolower($1)], [$7])])])],
     with_$1=$withval, with_$1=yes)
 
   ac_octave_$1_pkg_check=no
   m4_toupper([$1])_LIBS=
   warn_$1="$3"
   case $with_$1 in
     no)
       ifelse([$#], 10,
-        [AC_MSG_ERROR([--without-$1 specified but $2 is required.])],
-        [warn_$1="--without-$1 specified.  Functions or features that depend on $2 will be disabled."
+        [AC_MSG_ERROR([--without-m4_tolower($1) specified but $2 is required.])],
+        [warn_$1="--without-m4_tolower($1) specified.  Functions or features that depend on $2 will be disabled."
          m4_toupper([$1])_LIBS=])
     ;;
     yes | "")
       ac_octave_$1_pkg_check=yes
       m4_toupper([$1])_LIBS="-l$1"
     ;;
     -* | */* | *.a | *.so | *.so.* | *.o)
       m4_toupper([$1])_LIBS="$with_$1"
